dynamic-plugins/wrappers/red-hat-developer-hub-backstage-plugin-global-header/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "red-hat-developer-hub-backstage-plugin-global-header",
-  "version": "1.6.0",
+  "version": "1.7.1",
```

Comment:
Should this be 1.8.0? 

https://www.npmjs.com/package/@red-hat-developer-hub/backstage-plugin-global-header/v/1.8.0

Reply:
We can use the latest which is 1.9.0 now
https://www.npmjs.com/package/@red-hat-developer-hub/backstage-plugin-global-header